### ISSUE TITLE
Bot keeps spamming games. Please take it down.

### ISSUE THREAD

**Issue by samfin** [alekxeyuk/Skribbl.io-Bot#3]

**Comment by kylejcatterall** [comment]
If you wish to contact the owner on matters such as taking it down for ethical issues, perhaps you should provide any contact information. Nobody needs debates in the middle of issue tabs.

**Comment by SohaibJKhan** [comment]
No. There's spam all over the game. You're just being annoying.
